docs/content.zh/docs/dev/table/sql/show.md

```diff
@@ -511,4 +547,15 @@ SHOW JARS
 
 <span class="label label-danger">Attention</span> 当前 SHOW JARS 命令只能在 [SQL CLI]({{< ref "docs/dev/table/sqlClient" >}}) 中使用。
 
+## SHOW COLUMNS
+
+```sql
+SHOW COLUMNS ( FROM | IN ) <table_name> [ [NOT] LIKE <sql_like_pattern>]
+```
+
+展示给定表的所有列。
+
+**LIKE**
+根据可选的 `LIKE` 语句展示给定表中与 `<sql_like_pattern>` 是否模糊相似的所有列。
```

Comment:
Could you add more examples to explain how to use the LIKE clause? I think Hive gives a good example.